include/fmt/format.h

```diff
@@ -2124,13 +2124,14 @@ auto write_int(OutputIt out, UInt value, unsigned prefix,
                const format_specs<Char>& specs,
                const digit_grouping<Char>& grouping) -> OutputIt {
   static_assert(std::is_same<uint64_or_128_t<UInt>, UInt>::value, "");
+  auto minimum = [](int a, int b) { return a < b ? a : b; };
   int num_digits = 0;
-  char digits[40];
+  char digits[40]; 
 
   switch (specs.type) {
   case presentation_type::none:
   case presentation_type::dec: {
-    num_digits = std::min(count_digits(value), 40);
```

Comment:
Replace ```std::min``` with ```(std::min)```.

Reply:
Thank you for the comment, I ended up not needing `std::min` since I replaced the `digits` array.